src/scribe_data/cli.py

```diff
@@ -19,3 +19,178 @@
+    elif word_type == 'emoji_keywords':
+        max_key_length = max(len(key) for key in data.keys())
+        for key, value in data.items():
+            emojis = [item['emoji'] for item in value]
+            print(f"{key:<{max_key_length}} : {' '.join(emojis)}")
+    elif word_type == 'prepositions' or word_type == 'translations':
+        max_key_length = max(len(key) for key in data.keys())
+        for key, value in data.items():
+            print(f"{key:<{max_key_length}} : {value}")
+    else:
+        if isinstance(data, dict):
+            max_key_length = max(len(key) for key in data.keys())
+            for key, value in data.items():
+                if isinstance(value, dict):
+                    print(f"{key:<{max_key_length}} : ")
+                    max_sub_key_length = max(len(sub_key) for sub_key in value.keys())
+                    for sub_key, sub_value in value.items():
+                        print(f"  {sub_key:<{max_sub_key_length}} : {sub_value}")
+                elif isinstance(value, list):
+                    print(f"{key:<{max_key_length}} : ")
+                    for item in value:
+                        if isinstance(item, dict):
+                            for sub_key, sub_value in item.items():
+                                print(f"  {sub_key:<{max_key_length}} : {sub_value}")
+                        else:
+                            print(f"  {item}")
+                else:
+                    print(f"{key:<{max_key_length}} : {value}")
+        elif isinstance(data, list):
+            for item in data:
+                if isinstance(item, dict):
+                    for key, value in item.items():
+                        print(f"{key} : {value}")
+                else:
+                    print(item)
+        else:
+            print(data)
+
+def query_data(language: str, word_type: str) -> None:
+    # Normalize the input language
+    normalized_language = LANGUAGE_MAP.get(language.lower())
+    if not normalized_language:
+        print(f"Language '{language}' is not recognized.")
+        return
+
+    data_file = DATA_DIR / normalized_language / f"{word_type}.json"
+    if not data_file.exists():
+        print(f"No data found for language '{normalized_language}' and word type '{word_type}'.")
+        return
+
+    try:
+        with data_file.open('r') as file:
+            data = json.load(file)
+    except (IOError, json.JSONDecodeError) as e:
+        print(f"Error reading '{data_file}': {e}")
+        return
+
+    print(f"Data for language '{normalized_language}' and word type '{word_type}':")
+    print_formatted_data(data, word_type)
+
+    if word_type.lower() == 'nouns':
+        print("\nLegend:")
+        print("PL    : Plural")
+        print("empty : Singular\n")
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description='Scribe-Data CLI Tool')
+    subparsers = parser.add_subparsers(dest='command', required=True)
```

Comment:
I think it's fine if we just go ahead and add all the commands that we already thought of here (#136). We can leave them with no implementation for now, but just so we can get the basic command structure started. 

ooh an idea could also be to simply have a generic `def not_implemented()` function that prints out a message stating that the command is not implemented yet. Then we can just have all the commands call that for now :thinking: 

:round_pushpin: